<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 823, in lint_file
    recipe = _recipe.Recipe.from_file(
        recipe_fname=str(meta_yaml),
    ...<2 lines>...
        renderer=RendererType.RUAMEL,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 245, in from_file
    raise exc from exc
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 236, in from_file
    recipe._load_from_string(text.read())
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 172, in _load_from_string
    self.render()
    ~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 297, in render
    self.meta = renderer_utils.render(self.recipe_file, self.dump(), self.selector_dict, self.renderer)
                ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/_renderer.py", line 263, in render
    return ruamel.load(yaml_text.replace("\t", " ").replace("%", " "))
           ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/main.py", line 451, in load
    return constructor.get_single_data()
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/constructor.py", line 114, in get_single_data
    node = self.composer.get_single_node()
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 72, in get_single_node
    document = self.compose_document()
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 95, in compose_document
    node = self.compose_node(None, None)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 130, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 211, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 130, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 211, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 106, in compose_node
    if self.parser.check_event(AliasEvent):
       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/parser.py", line 141, in check_event
    self.current_event = self.state()
                         ~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/parser.py", line 631, in parse_block_mapping_value
    if self.scanner.check_token(ValueToken):
       ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 1735, in check_token
    self._gather_comments()
    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 1775, in _gather_comments
    self.fetch_more_tokens()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 278, in fetch_more_tokens
    return self.fetch_value()
           ~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 628, in fetch_value
    raise ScannerError(
    ...<4 lines>...
    )
ruamel.yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 9, column 11:
      url: url: https://pypi.org/packages/sour ...
              ^ (line: 9)

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_7fumj4arra/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
 
 source:
-  url: url: https://pypi.org/packages/source/p/panel_graphic_walker/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/p/panel_graphic_walker/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
   sha256: 0881df1300e17a63ada0c6689f57ddb3f435c84576278a0f6092ed1239dee9ad
 
 build:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
 
 source:
-  url: url: https://pypi.org/packages/source/p/panel_graphic_walker/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  url: url: https://pypi.org/packages/source/p/panel_graphic_walker/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
   sha256: 0881df1300e17a63ada0c6689f57ddb3f435c84576278a0f6092ed1239dee9ad
 
 build:

